packages/ui/src/components/Tooltip.tsx

```diff
@@ -0,0 +1,332 @@
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useReducer,
+  useRef,
+  useState,
+} from "react";
+import {
+  ColorValue,
+  StyleProp,
+  StyleSheet,
+  View,
+  ViewStyle,
+} from "react-native";
+
+const styles = StyleSheet.create({
```

Comment:
You did the right thing by just taking a stab here. I'll come back through and match the art direction once we get everything squared away.